source/minimessage/format.rst

```diff
@@ -0,0 +1,272 @@
+   * ``_colorNameOrHex_``, can be any of the values from above (so named colors or hex colors)
+Examples
+   * :mm:`<color:yellow>Hello <color:blue>World</color:blue>!`
+   * :mm:`<color:#FF5555>This is a <color:#55FF55>test!`
+
+.. image:: /minimessage/images/color_verbose_1.png
+   :alt: The result of parsing ``<color:yellow>Hello <color:blue>World</color:blue>!``, shown in-game in the Minecraft client's chat window
+
+.. image:: /minimessage/images/color_verbose_2.png
+   :alt: The result of parsing ``<color:#FF5555>This is a <color:#55FF55>test!``, shown in-game in the Minecraft client's chat window
+
+Decoration
+***********
+
+Decorate the next parts
+
+Tag
+   :mm:`<_decorationname_[:false]>`, or :mm:`<!_decorationname_>` as an alias to invert the decoration.
+Arguments:
+   ``_decorationname_``, Any decoration supported in Minecraft:
+   
+   =================   =======
+   Decoration           Aliases
+   =================   =======
+   ``bold``            ``b``
+   ``italic``          ``em`` or ``i``
+   ``underlined``      ``u``
+   ``strikethrough``   ``st``
+   ``obfuscated``      ``obf``
+   =================   =======
+
+Examples:
+   * :mm:`<underlined>This is <bold>important</bold>!`
+
+.. image:: /minimessage/images/decoration_1.png
+   :alt: The result of parsing ``<underlined>This is <bold>important</bold>!``, shown in-game in the Minecraft client's chat window
+
+Reset
+*****
+
+Close all currently open tags, resetting colour/decoration/etc. The reset tag cannot be closed.
+
+In strict mode, reset tags are forbidden.
+
+Tag
+   :mm:`<reset>`
+Aliases
+   ``r``
+Arguments
+   non
+Examples
+   * :mm:`<yellow><bold>Hello <reset>world!`
+
+.. image:: /minimessage/images/reset_1.png
+   :alt: The result of parsing ``<yellow><bold>Hello <reset>world!``, shown in-game in the Minecraft client's chat window
+
+Click
+*****
+
+Allows doing multiple things when clicking on the component.
+
+Tag
+   :mm:`<click:_action_:_value_>`
+Arguments
+   * ``_action_``, the type of click event, one of `this list <https://jd.adventure.kyori.net/api/latest/net/kyori/adventure/text/event/ClickEvent.Action.html#enum.constant.summary>`_
+   * ``_value_``, the argument for that particular event, refer to `the minecraft wiki <https://minecraft.gamepedia.com/Raw_JSON_text_format>`_
+Examples
+   * :mm:`<click:run_command:/say hello>Click</click> to say hello`
+   * :mm:`Click <click:copy_to_clipboard:Haha you suck> this </click>to copy your score!`
+
+.. image:: /minimessage/images/click_1.png
+   :alt: The result of parsing ``<click:run_command:/say hello>Click</click> to say hello``, shown in-game in the Minecraft client's chat window
+
+Hover
+*****
+
+Allows doing multiple things when hovering on the component.
+
+Tag
+   :mm:`<hover:_action_:_value..._>`
+Arguments
+   * ``_action_``, the type of hover event, one of this `list <https://jd.adventure.kyori.net/api/latest/net/kyori/adventure/text/event/HoverEvent.Action.html#field.summary>`_
```

Comment:
Can't speak for others but I will say from personal experience when I was reading the minimessage docs for the first time, linking to a javadoc was very confusing for something which isn't really API documentation. (also linking to a specification for JSON chat probably doesn't fit that much either?)
Is it overly verbose to just list the options?
But also one of them has `this list` as the link while the other has just `list`

Reply:
I would like to eliminate all references to the Minecraft wiki's JSON text page, and to Javadoc -- but the original version of this documentation linked directly to Adventure source so I thought that Javadoc would be a step forward.

Reply:
Oh right, it used to link to source!! I sorta remembered it being worse than just javadoc lol